<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>292</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Chartwork Installer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="installButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>246</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeQtDirectoryButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Qt Directory</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>631</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>192</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/installer.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="qtPathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>207</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Qt path:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qtCreatorPathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>225</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Qt Creator path:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qtPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>207</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\Qt\Qt5.3.2\5.3\msvc2013\</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qtCreatorPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>225</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\Qt\Qt5.3.2\Tools\QtCreator\</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>image</zorder>
   <zorder>installButton</zorder>
   <zorder>changeQtDirectoryButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>qtPathLabel</zorder>
   <zorder>qtCreatorPathLabel</zorder>
   <zorder>qtPath</zorder>
   <zorder>qtCreatorPath</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
